docs/installation.rst

```diff
@@ -1,5 +1,145 @@
 ============
 Installation
 ============
+Installation documentation for Noggin.
 
-.. note:: **TODO**: Cover end-user installation here.
+
+============
+Requirements
+============
+Noggin requires the following to operate:
+
+1. Access to a FreeIPA/Red Hat Identity Management installation
+2. Access to an Openshift cluster to run the Noggin container
+
+
+===================================
+Install Red Hat Identity Management
+===================================
+For more information see the following documents regarding installation
+
+1. .. _Docs: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/installing_identity_management/preparing-the-system-for-ipa-server-installation_installing-identity-management
+2. .. _Playbook install docs: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/installing_identity_management/installing-an-identity-management-replica-using-an-ansible-playbook_installing-identity-management
+
+
+Install ansible and ansible-freeipa rpms
+
+::
+    subscription-manager repos --enable ansible-2.8-for-rhel-8-x86_64-rpms
+    yum install ansible
+    yum install ansible-freeipa
+
+
+The location where the ansible roles are installed:
+
+::
+    [root@server]# ls -1 /usr/share/ansible/roles/
+    ipaclient
+    ipareplica
+    ipaserver
+
+
+The location where the documentation is once installed:
+
+::
+    [root@server]# ls -1 /usr/share/doc/ansible-freeipa/
+    playbooks
+    README-client.md
+    README.md
+    README-replica.md
+    README-server.md
+    README-topology.md
+
+
+The location where the ansible playbooks are once installed:
+
+::
+    [root@server]# ls -1 /usr/share/doc/ansible-freeipa/playbooks/
+    install-client.yml
+    install-cluster.yml
+    install-replica.yml
+    install-server.yml
+    uninstall-client.yml
+    uninstall-cluster.yml
+    uninstall-replica.yml
+    uninstall-server.yml
+
+
+A sample playbook making use of the role to install the server:
+
+::
+    [root@server]# cat install_server.yaml
+    ---
+    - name: Playbook to configure IPA server
+      hosts: ipaserver
+      become: true
+
+      roles:
+      - role: ipaserver
+        state: present
+
+    [root@server]# ansible-playbook -i inventory install_server.yaml
```

Comment:
The installation instructions of FreeIPA didn't have to be in there, but nice that they are. However, I'm missing instructions on creating service accounts (with minimal privileges), and for example freeipa-fas.

Reply:
@puiterwijk I think I've addressed this, can you take another look please!